examples/ReorderExample.js

```diff
@@ -44,7 +28,7 @@ class ReorderExample extends React.Component {
     super(props);
 
     this.state = {
-      dataList: new FakeObjectDataListStore(1000000),
+      dataList: new FakeObjectDataListStore(10000),
```

Comment:
I did this because this example is really jank with dev React (but fine with prod React).  I can revert though if desired.